src/components/ParticipantManifestModal.js

```diff
@@ -24,7 +25,22 @@ const ParticipantManifestModal = ({
           projectId,
           setWarning,
           onDownloadClick: async () => {
-            await fileManifestParticipantsOnly({ sqon, columns })();
+            await fileManifestParticipantsOnly({ sqon, columns })()
```

Comment:
same q here re: moving into download function itself

Reply:
I believe this file is no longer referenced as both the two download modal have been merged to one

Reply:
oh yeah true, was this a merge mishap? or was it there / we just forgot to remove it?

Reply:
I think this was a merge mishap, I saw it and didn't want to remove because I hadn't looked at Minh's modal updates